src/main/java/com/sap/cloudfoundry/client/facade/domain/ServicePlanVisibility.java

```diff
@@ -0,0 +1,16 @@
+package com.sap.cloudfoundry.client.facade.domain;
+
+public enum ServicePlanVisibility {
+
+    PUBLIC("public"), ADMIN("admin"), ORGANIZATION("organization");
```

Comment:
Since the values are the enum values but to lowercase, remove the `value` field and method, and override `toString` to return `name().toLowerCase()`

Reply:
Done